.github/workflows/test.yml

```diff
@@ -192,6 +192,8 @@ jobs:
             **/go.sum
             **/Makefile
             Makefile
+            Dockerfile
+            Dockerfile.gaia
```

Comment:
Maybe do Dockerfile
and Dockerfile.*
e.g. if we get a Dockerfile.test at some point, it's covered.
(or even just Dockerfile*, instead)